src/awkward/_v2/_util.py

```diff
@@ -23,6 +24,14 @@
 
 np = ak.nplike.NumpyMetadata.instance()
 
+py27 = sys.version_info[0] < 3
+py35 = sys.version_info[0] == 3 and sys.version_info[1] <= 5
+
+# to silence flake8 F821 errors
+if py27:
+    unicode = eval("unicode")
+else:
+    unicode = None
```

Comment:
Awkward v2 isn't supporting Python 2.7 or 3.5, so these won't be needed for anything.

(All of the Python 2.7 checks at the beginning of each v2 test will be removed when Python 2.7 is removed from CI.)